man/journald.conf.xml

```diff
@@ -25,6 +25,7 @@
   <refsynopsisdiv>
     <para><simplelist>
       <member><filename>/etc/systemd/journald.conf</filename></member>
+      <member><filename>/usr/lib/systemd/journald.conf</filename></member>
```

Comment:
Please also add `/run/systemd/journald.conf`.

Reply:
Added in new revision.